### PR TITLE
add payout recipient to settings, default to claim user, fix #326

### LINKED ISSUE #326: Update PayPal payout to use claim.user.email
When a user claims their payout, send the PayPal payout to their claim.user.email, so they can really get the payout.

*Note*: When an environment variable, `PAYPAL_PAYOUT_RECIPIENT`, is set, send the PayPal payouts to that recipient.

### PR DESCRIPTION
**Pull request by jdungan**
Adds PAYPAL_PAYOUT_RECIPIENT fto .env and references it in models.

When PAYPAL_PAYOUT_RECIPIENT is set it will override the payout recipient on payout requests. 

Otherwise, claim.user.email is now used for sending payouts.

### CODE REVIEW

File: auctions/models.py [hunk 1]
```diff
@@ -28,6 +28,8 @@
                          "client_secret": settings.PAYPAL_CLIENT_SECRET,
                          })
 
+paypay_recipient = settings.PAYPAL_PAYOUT_RECIPIENT
```
Comment (groovecoder) [inline auctions/models.py]: typo: `paypay` also conventional to put these module-level constants in all caps: `PAYPAL_RECIPIENT`

File: auctions/models.py [hunk 2]
```diff
@@ -479,7 +481,9 @@ def fees(self):
         return PayoutFee.objects.filter(payout=self)
 
     def request(self):
-
+        receiver = (
+            paypay_recipient if paypay_recipient else self.claim.user.email
```
Comment (groovecoder) [inline auctions/models.py]: `paypal_` ... you're subconsciously typing "pay" so many times - I can tell you're very excited about the payments.

File: auctions/models.py [hunk 3]
```diff
@@ -28,6 +28,8 @@
                          "client_secret": settings.PAYPAL_CLIENT_SECRET,
                          })
 
+PAYPAL_PAYOUT_RECIPIENT = settings.PAYPAL_PAYOUT_RECIPIENT
```
Comment (groovecoder) [inline auctions/models.py]: Can we skip this extra assignment and just use `settings.PAYPAL_PAYOUT_RECIPIENT` below?

Comment (jdungan) [inline auctions/models.py]: Thought the settings assignments should be upfront.  Especially if we use this on other models in the future.

Comment (groovecoder) [inline auctions/models.py]: Yeah, that's fine.

**Pull request merged**